debian/jitsi-meet-web-config.postinst

```diff
@@ -95,7 +95,6 @@ case "$1" in
         JITSI_MEET_CONFIG="/etc/jitsi/meet/$JVB_HOSTNAME-config.js"
         if [ ! -f $JITSI_MEET_CONFIG ] ; then
             cp /usr/share/doc/jitsi-meet-web-config/config.js $JITSI_MEET_CONFIG
-            sed -i "s/jitsi-meet.example.com/$JVB_HOSTNAME/g" $JITSI_MEET_CONFIG
```

Comment:
@damencho what are your thoughts on this? On one had it makes the setup multi-domain friendly, OTOH, more setup would be required for that (AFAICT) so I'd rather have a sed there...